recipes/ppscore/LICENSE.txt

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
The PyPI tarball ships the license, so you can just remove this file from the PR.

Reply:
@dopplershift all set